### PR TITLE
{toolchain}[dummy/dummy] CrayGNU v2016.03

### PR DESCRIPTION
**Pull request by gppezzi**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/c/CrayGNU/CrayGNU-2016.03.eb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+easyblock = 'CrayToolchain'
+
+name = 'CrayGNU'
+version = '2016.03'
+
+homepage = 'http://docs.cray.com/books/S-9408-1603/'
+description = """Toolchain using Cray compiler wrapper, using PrgEnv-gnu module (PE release: November 2015).\n"""
```
Comment (boegel) [inline easybuild/easyconfigs/c/CrayGNU/CrayGNU-2016.03.eb]: this should say March 2016?

**Comment by gppezzi** [comment]
Test report by @gppezzi
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
dora103 - Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2650 v3 @ 2.30GHz, Python 2.7.7
See https://gist.github.com/8ced483eab90ba3d4d556cb80a92a126 for a full test report.

**Comment by boegel** [comment]
lgtm :+1:

**Comment by boegel** [comment]
Going in, thanks @gppezzi!

**Pull request merged**